src/theme/NotFound.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { NotFoundBlock } from '../components/notFoundBlock';
+
+export default () => {
```

Comment:
You can shorten it if it's not autogenerated code `export default NotFoundBlock `